blocks/core/ff_module/ff_module-form-preview/ff_module-form-preview.js

```diff
@@ -14,6 +15,7 @@ module.exports = React.createClass({
 				value: React.PropTypes.string,
 				modifier: React.PropTypes.string,
 				previewFor: React.PropTypes.string,
+				fileList: React.PropTypes.object,
```

Comment:
Might be a good idea to point to the propTypes definition in the FileList component itself here (with a `shape` wrapper) but not strictly necessary